cli/src/plugins/javadoc/index.ts

```diff
@@ -202,6 +215,16 @@ function makeSection(args: MakeSectionArgs): Node[] {
   ];
 }
 
+function getClassType(doc: ParsedClassDoc): string {
```

Comment:
Rather than string, I would define a specific enum called ClassType (`type ClassType = "error" | "exception" | "class"`) and return that.